<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TestForm</class>
 <widget class="QMainWindow" name="TestForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
      </layout>
      <widget class="QListWidget" name="listWidget" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text" >
         <string>Zegar</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Barometr</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Wykres</string>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page" >
           <widget class="QLabel" name="clock_lab" >
            <property name="geometry" >
             <rect>
              <x>10</x>
              <y>10</y>
              <width>111</width>
              <height>16</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Tutaj będzie zegar</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page" >
           <widget class="QLabel" name="bar_label" >
            <property name="geometry" >
             <rect>
              <x>10</x>
              <y>10</y>
              <width>161</width>
              <height>16</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Tutaj będzie barometr</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page" >
           <widget class="QLabel" name="qraph_lab" >
            <property name="geometry" >
             <rect>
              <x>10</x>
              <y>10</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Tutaj będzie wykres</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page" />
          <widget class="QWidget" name="page" />
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QSlider" name="Hslider" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QStackedWidget" name="stackedWidget_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy>
      <hsizetype>7</hsizetype>
      <vsizetype>7</vsizetype>
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="page" />
    <widget class="QWidget" name="page" />
    <widget class="QWidget" name="page" />
   </widget>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>Hslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>293</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>442</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>436</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel" >
     <x>251</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>252</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
